### ISSUE TITLE
Button: fix inferring alignment from HTML

### ISSUE THREAD

**Issue by mtias** [WordPress/gutenberg#653]
Waiting on #624

**Comment by aduth** [comment]
I believe this is closed by 3e144398a27cf48db1209d7aaec7e721aa26a9b9, though I think we might want to open another issue relating to avoiding duplication on alignment between block attributes and classes, leaning toward `data-` attributes for both parsing and styling alignment.
